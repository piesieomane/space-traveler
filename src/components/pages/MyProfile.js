import MissionProfile from './MissionProfile';
import RocketProfile from './RocketProfile';

const MyProfile = () => (
  <div className="profile-flex">
    <div>
      <h1>My Rockets</h1>
      <RocketProfile />
    </div>
    <div>
      <h1>My Missions</h1>
      <MissionProfile />
    </div>
  </div>
);
export default MyProfile;
