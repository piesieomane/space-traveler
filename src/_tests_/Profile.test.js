import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import MyProfile from '../components/pages/MyProfile';

it('renders correctly', () => {
  const tree = Renderer.create(
    <React.StrictMode>
      <Provider store={store}>
        <BrowserRouter>
          <MyProfile />
        </BrowserRouter>
      </Provider>
    </React.StrictMode>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
