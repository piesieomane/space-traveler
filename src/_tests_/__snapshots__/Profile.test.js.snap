// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="profile-flex"
>
  <div>
    <h1>
      My Rockets
    </h1>
    <div>
      <table
        className="styled-table"
      >
        <tbody />
      </table>
    </div>
  </div>
  <div>
    <h1>
      My Missions
    </h1>
    <div>
      <table
        className="styled-table"
      >
        <tbody />
      </table>
    </div>
  </div>
</div>
`;
